examples/simulation/TensorBoardStrategy.py

```diff
@@ -0,0 +1,58 @@
+import flwr as fl
+import tensorflow as tf
+from typing import Optional
+from typing import List, Dict, Optional, Tuple
+from flwr.common import Scalar
+import os, os.path
+
+
+def weighted_loss_avg(results: List[Tuple[int, float, Optional[float]]]) -> float:
+    """Aggregate evaluation results obtained from multiple clients."""
+    num_total_evaluation_examples = sum(
+        [num_examples for num_examples, _, _ in results]
+    )
+    weighted_losses = [num_examples * loss for num_examples, loss, _ in results]
+    return sum(weighted_losses) / num_total_evaluation_examples
+
+
+class TensorBoardStrategy(fl.server.strategy.FedAvg):
+    def aggregate_evaluate(
+        self, rnd: int, results, failures,
+    ) -> Tuple[Optional[float], Dict[str, Scalar]]:
+        """Aggregate evaluation losses using weighted average."""
+        loss_aggregated = super().aggregate_evaluate(rnd, results, failures)[0]
+        writer_distributed = tf.summary.create_file_writer("mylogs/distributed")
+
+        if rnd != -1:
+            step = rnd
+        else:
+            step = len(
+                [
+                    name
+                    for name in os.listdir("mylogs/distributed")
+                    if os.path.isfile(os.path.join("mylogs/distributed", name))
```

Comment:
Might be cool if this is configurable and defaults to e.g. `$(cwd)/tensorboard`

Reply:
yes this was just a hack since I was not sure how to consider rnd=-1. We can definitely do it in  a cleaner way :)